cmd/vdev_id/vdev_id

```diff
@@ -276,6 +276,26 @@ sas_handler() {
 		d=$(eval echo \${$i})
 		SLOT=`echo $d | sed -e 's/^.*://'`
 		;;
+	"SES")
+		# look for this SAS path in all SCSI Enclosure Services (SES) enclosures
+		sas_address=`cat $end_device_dir/sas_address 2>/dev/null`
+		# example output of lsscsi -g:
+		# [1:0:25:0]  enclosu HP       D2700 SAS AJ941A 0147  -          /dev/sg27
+		# [4:0:25:0]  enclosu HP       D2700 SAS AJ941A 0147  -          /dev/sg53
+		# [4:0:51:0]  enclosu HP       D2700 SAS AJ941A 0147  -          /dev/sg79
+		# [5:0:25:0]  enclosu HP       D2700 SAS AJ941A 0147  -          /dev/sg105
+		# [5:0:61:0]  enclosu HP       D6000            2.38  -          /dev/sg141
+		# [6:0:25:0]  enclosu HP       D2700 SAS AJ941A 0147  -          /dev/sg167
+		# [6:0:51:0]  enclosu HP       D2700 SAS AJ941A 0147  -          /dev/sg193
+		enclosures=`lsscsi -g | sed -n -e '/enclosu/s/^.* \([^ ]*\)$/\1/p'`
```

Comment:
The sed line here doesn't appear to correctly handle the following abbreviated output from `lsscsi -g`.  Specifically it does not print the first device.

```
[0:0:0:0]    enclosu RAIDINC  84BAY EBOD       0204  -          /dev/sg0 
[0:0:81:0]   enclosu RAIDINC  84BAY EBOD       0204  -          /dev/sg81
[11:0:1:0]   enclosu RAIDINC  84BAY EBOD       0204  -          /dev/sg163
[11:0:81:0]  enclosu RAIDINC  84BAY EBOD       0204  -          /dev/sg243
```

```sh
lsscsi -g | sed -n -e '/enclosu/s/^.* \([^ ]*\)$/\1/p'

/dev/sg81
/dev/sg163
/dev/sg243
```

Reply:
Fixed to cope with the pesky trailing space.